### PR TITLE
Merge release changes back into main

### PR DESCRIPTION
**Pull request by rchiodo**
According to the release [plan](https://github.com/microsoft/vscode-jupyter/issues/7909) this is moving changelog items over.

Engine version will change later when we turn insiders back on.

From this point on, we're shipping only release.

### CODE REVIEW

**Comment by IanMatthewHuff** [comment]
I can't quite remember the process, but do you want to bump the engine now (1.63-insiders or whatever) so that we don't accidentally release an insiders build that overwrites live stuff when we turn things back on?

**Comment by rchiodo** [comment]
> I can't quite remember the process, but do you want to bump the engine now (1.63-insiders or whatever) so that we don't accidentally release an insiders build that overwrites live stuff when we turn things back on?

No we instead turn off the insiders version. We don't bump until we ship the release bits so that it can continue to run with the insiders we have on our own boxes.

See the entry in the release plan:
https://github.com/microsoft/vscode-jupyter/issues/7909

**Review by DonJayamanne — APPROVED** [review]

**Review by IanMatthewHuff — APPROVED** [review]

File: package.json [hunk 1]
```diff
@@ -28,7 +28,7 @@
         "theme": "light"
     },
     "engines": {
-        "vscode": "1.62.0-insider"
```
Comment (DonJayamanne) [inline package.json:31]: I thought this should be 1.63.0-insider

Comment (DonJayamanne) [reply package.json:31]: Perhaps not, i guess its ok. Can't remember what i did.
It shouldn't matter as releases are off

**Pull request merged**